common/components/timeline/_timeline.scss

```diff
@@ -2,79 +2,82 @@
    #TIMELINE
    ========================================================================== */
 
-   .app-timeline {
-    margin-bottom: govuk-spacing(4);
-  }
+.app-timeline {
```

Comment:
Are there any changes to this file? It's a little hard to see with the whitespace noise.

Reply:
Yep. Should be clearer now.

Reply:
Really should be clearer now ;)